README.md

```diff
@@ -20,7 +20,9 @@ Update Foreman with the new gems:
 # Configuration
 
 By default the plugin will display the Domain associated by each host. This is not
-massively useful. To set your own choice of column, add this to Foreman's config file
+massively useful. To set your own choice of column, add this to Foreman's plugin config file 
+foreman_column_view.yaml. For package based installs this should be in 
+/etc/foreman/plugins/foreman_column_view.yaml. For source installs the directory may differ.
 
```

Comment:
Can you make this "For source installs, this should be in config/settings.plugins.d within your install directory". Also, using backticks around paths will help readability, eg \'foreman_column_view.yaml\'